templates/packer/ansible/roles/common/tasks/packages.yml

```diff
@@ -0,0 +1,76 @@
+---
+
+- name: Add osbuild-composer repository
+  yum_repository:
+    name: "composer"
+    description: "osbuild-composer commit {{ COMPOSER_COMMIT }}"
+    baseurl: "http://osbuild-composer-repos.s3.amazonaws.com/osbuild-composer/rhel-8-cdn/x86_64/{{ COMPOSER_COMMIT }}"
+    enabled: yes
+    gpgcheck: no
+    priority: "5"
+
+- name: Add osbuild repository
+  yum_repository:
+    name: "osbuild"
+    description: "osbuild commit {{ OSBUILD_COMMIT }}"
+    baseurl: "http://osbuild-composer-repos.s3-website.us-east-2.amazonaws.com/osbuild/rhel-8-cdn/x86_64/{{ OSBUILD_COMMIT }}"
+    enabled: yes
+    gpgcheck: no
+    priority: "5"
+
+# We need EPEL for monit
+- name: Add EPEL
+  dnf:
+    name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+    state: present
+    disable_gpg_check: yes
+
+- name: Upgrade all packages
+  package:
+    name: "*"
+    state: latest
+  register: result
+  retries: 5
+  until: result is success
+
+- name: Add Vector repo
+  copy:
+    src: "{{ playbook_dir }}/roles/common/files/timber-vector.repo"
+    dest: /etc/yum.repos.d/
+
+- name: Install required packages
+  package:
+    name:
+      - jq
+      - osbuild-composer-worker
+      - unzip
+      - monit
+      - vector
+  register: result
+  retries: 5
+  until: result is success
+
+- name: Download AWS CLI installer
```

Comment:
If we need this, is it not available from a yum repo?

Reply:
officially this is the way. Also v2 (which this link points to) is fully standalone.

Reply:
Yeah, I would honestly follow the official docs when possible.